build.gradle

```diff
@@ -53,8 +76,8 @@ ext {
 	]
 
 	// Code Coverage (Lines/Branches) cannot be below this value on a per sub project basis
-	minUnitTestLineCoverage = 0.8
-	minUnitTestBranchCoverage = 0.8
+	minUnitTestLineCoverage = 0.9
+	minUnitTestBranchCoverage = 0.9
```

Comment:
Do we need this to be at 90%?

Reply:
It only needed lowered due to an unnecessary check in IssueCredential. Confirm with AC tests. The actual branch coverage in IssueCredential is 100 now :), but I was thinking that was a little high to maintain 📈

Reply:
Can we keep at 80 if thats all that is required? 90 is fine but don't want a later pr including a downgrade in test coverage to make us look bad